frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/ERDContent.tsx

```diff
@@ -181,6 +181,8 @@ export const ERDContent: FC<Props> = ({
     [setEdges],
   )
 
+  const panOnDrag = [1, 2]
```

Comment:
[1, 2] allows panning with the middle and right mouse buttons.

https://github.com/xyflow/xyflow/blob/0f21ec21ebf01567ec1de4fad1d0c55f40db3463/packages/react/src/types/component-props.ts#L350-L356